src/style.css

```
@@ -1 +1,97 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  line-height: 14px;
+  font-size: 12px;
+  color: #000;
+}
+
+.header {
+  display: flex;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: nowrap;
+  height: 60px;
+  padding: 0 50px;
+
+}
+
+.nav__list {
+  padding: 0;
+}
+
+.image {
+  width: 40px;
+  height: 40px;
+    
+}
+
+.nav {
+  margin: 0 50px;
+  /* padding: 0 10px; */
+}
+
+.nav__item {
+  display: inline-block;
+  padding-left: 20px;
```

use `margin` instead
```suggestion
  margin-left: 20px;
```